deicode/q2/_method.py

```diff
@@ -6,10 +6,10 @@
 from deicode.optspace import OptSpace
 from deicode.preprocessing import rclr
 
-def rpca_biplot(table: biom.Table, rank: int=3, 
+def rpca(table: biom.Table, rank: int=3, 
```

Comment:
Looks like flake8 is not running as part of the CI build, would be good to include that in a dedicated PR.

Reply:
added this to travis! very helpful.